src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -34,35 +34,9 @@ import infrastructureWindows from 'images/infrastructure_logo_windows.png'
+Forwarding your logs to New Relic makes all of your logging data available in one location, providing deeper visibility into both your application and your platform performance data. With your logs in one spot, you get enhanced log management capabilities that let you collect, process, explore, query, and alert on errors or anomalies found in your log data. 
 
-Forwarding your logs to New Relic will give you enhanced log management capabilities to collect, process, explore, query, and alert on your log data. To see how logs in context of your apps and hosts can help you find the root cause of an issue, watch this short video (approx. 3:40 minutes):
-
-<Video
-  id="Vgae-GrlZWg"
-  type="youtube"
-/>
-
-## Basic process [#enable]
-
-For an overview of all the options available for New Relic log management, see [Introduction to logs](/docs/logs/get-started/get-started-log-management#integrate-logs). 
-
-When it comes to reporting logs with our infrastructure agent, there are two main routes:
-
-* Use our [guided install](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview), which is a lightweight install route that you can use to install our infrastructure agent, an APM agent, and more. The guided install is a good choice for smaller organizations or any organization that wants to test out New Relic. 
-* Do a standard manual install of our infrastructure agent. This will require more work than the guided install but will unlock more power. 
-
-Here are the steps for setting up log reporting via our infrastructure monitoring: 
-
-1. If you haven't already, [create a New Relic account](https://newrelic.com/signup). It's free, forever.
-2. Verify the [system requirements](#requirements) needed for configuring logs.
-3. Ensure you have [installed the infrastructure agent](#install), version 1.11.4 or higher. (If you use the guided install process in the UI, skip the following steps.)
-4. Create a `logging.yml` [configuration file](#configure) in the infrastructure agent's `logging.d` directory.
-5. [Configure](#parameters) your log sources and other parameters.
-6. Generate some traffic and wait a few minutes, then [check your account](#find-data) for data.
-7. [Explore your log data](#what-next) in the logs UI. In New Relic, look for log data that's been reported by the infrastructure agent.
-
-Here's an example of logs for your host's UI. You can see logs in context of events for the selected time period, and drill down into detailed data for any of the highlighted attributes. To examine even more detailed data, run a query, or click **Open in logs**.
+Your first step is to [create a New Relic account](). After that, use this doc to set up log forwarding. 
```

Comment:
Not sure if the link is supposed to be missing here, just doing due diligence so we have a marker.

Reply:
LOL omg thank you.